### PR TITLE
MP-6161 Marketplace Merchant Portal Security feature

### PR DESCRIPTION
**Pull request by volodymyr-bezuhlyi**
Tickets:
https://spryker.atlassian.net/browse/MP-6326

### CODE REVIEW

**Review by ilyakubanov — COMMENTED** [review]
We don't have a feature `Marketplace Merchant Portal Security`, we just updated some modules of already existed features. Please check what has been updated and adjust the corresponding IGs and feature walkthroughs.

File: docs/marketplace/dev/feature-integration-guides/202200.0/marketplace-merchant-portal-core-feature-integration.md [hunk 1]
```diff
@@ -239,6 +266,16 @@ class MerchantUserDependencyProvider extends SprykerMerchantUserDependencyProvid
...
         ];
     }
+    
+    /**
+     * @return array<\Spryker\Zed\MerchantUserExtension\Dependency\Plugin\MerchantUserRoleFilterPreConditionPluginInterface>
+     */
+    protected function getMerchantUserRoleFilterPreConditionPlugins(): array
+    {
+        return [
+            new AclMerchantPortalMerchantUserRoleFilterPreConditionPlugin(),
```
Comment (asmarovydlo) [inline docs/marketplace/dev/feature-integration-guides/202200.0/marketplace-merchant-portal-core-feature-integration.md:276]: Please add verification steps for newly added plugin**s**.

**Review by asmarovydlo — CHANGES_REQUESTED** [review]

File: docs/marketplace/dev/feature-integration-guides/202200.0/marketplace-merchant-portal-core-feature-integration.md (hunk 1 above)
Comment (volodymyr-bezuhlyi) [reply docs/marketplace/dev/feature-integration-guides/202200.0/marketplace-merchant-portal-core-feature-integration.md:282]: fixed

**Review by asmarovydlo — DISMISSED** [review]
Please resolve conflicts.

File: docs/marketplace/dev/feature-integration-guides/202204.0/marketplace-merchant-portal-core-feature-integration.md [hunk 2]
```diff
@@ -146,6 +148,36 @@ class SecurityDependencyProvider extends SprykerSecurityDependencyProvider
...
+        return [
+            new MerchantUserUserRoleFilterPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that Merchant Users or Users whose Acl Group does not have Backoffice allowed Acl Group Reference can not log in to Backoffice.
```
Comment (AdvaBD) [inline docs/marketplace/dev/feature-integration-guides/202204.0/marketplace-merchant-portal-core-feature-integration.md:177]: cannot log in

**Review by AdvaBD — CHANGES_REQUESTED** [review]

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**